__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -539,7 +539,8 @@ describe('TESTING ChannelListSchema', () => {
             id: 'lastUpdatedBy'
           }
         },
-        targetName: 'name'
+        targetName: 'name',
+        targetImage: ''
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `targetImage` property is a good move, but it's initialized with an empty string. If there's a default image URL for the target, consider initializing it with that to avoid potential issues with rendering an empty string as an image source.
```diff
-        targetImage: ''
+        targetImage: 'https://default-image-url.com/default.png'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->